### PR TITLE
Created basic api end points returning dummy data

### PR DESCRIPTION
**Pull request by christopher-gordon98**
- Added basic api end points

- Jquery to retreive end point data and set username

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,6 +21,76 @@ app.get('/account', function(request, response) {
...
+
+  switch (userId) {
+    case '01':
+      return response.json(toWatch1);
+      break;
+    case '02':
+      return response.json(toWatch2);
+      break;
+    default:
+      return response.json('list not found');
```
Comment (leeDav) [inline index.js]: If no user is found, you should also respond with a 404 ;)

File: index.js [hunk 2]
```diff
@@ -21,6 +21,76 @@ app.get('/account', function(request, response) {
...
+  let toWatch1 = {
+    data: ['back to the future2', 'star wars'],
+  };
+  let toWatch2 = {
+    data: ['star trek', 'game of thrones'],
+  };
+
+  switch (userId) {
+    case '01':
+      return response.json(toWatch1);
```
Comment (leeDav) [inline index.js]: Should return HTTP 200 when responding :)

File: index.js (hunk 1 above)
Comment (christopher-gordon98) [reply index.js]: Hi @leeDav  :)

This is something we definitely want to do in the future, however, for now we're trying to keep it as simple as possible and expand upon it later

Comment (christopher-gordon98) [reply index.js]: @leeDav Thank you for the suggestion :)

File: index.js (hunk 2 above)
Comment (christopher-gordon98) [reply index.js]: @leeDav Thank you for the suggestion :)

**Comment by aaron-goshine** [comment]
Looks good sofar based on the discussion we had. Please feel free to merge and we will refactor later.

**Pull request merged**